docs/sources/datasources/mysql/_index.md

```diff
@@ -297,7 +297,7 @@ The examples in this section query the following table:
 
 If the `Format as` query option is set to `Time Series` then the query must have a column named time that returns either a SQL datetime or any numeric datatype representing Unix epoch in seconds. In addition, result sets of time series queries must be sorted by time for panels to properly visualize the result.
 
-A time series query result is returned in a [wide data frame format](/developers/plugin-tools/introduction/data-frames#wide-format). Any column except time or of type string transforms into value fields in the data frame query result. Any string column transforms into field labels in the data frame query result.
+A time series query result is returned in a [wide data frame format](https://grafana.com/developers/plugin-tools/introduction/data-frames#wide-format). Any column except time or of type string transforms into value fields in the data frame query result. Any string column transforms into field labels in the data frame query result.
```

Comment:
@jdbaldry out of curiosity - why is this one an absolute link?

Reply:
That's an interesting question. Historically, the use of partial URLs was encouraged to be sympathetic to the staging site which has a different hostname. Now this is no longer necessary but there is the linter in place for some of these docs still insists on the use of partial URLs for now. This doc must be one that is not yet linted.